testfixtures/__init__.py

```diff
@@ -1,16 +1,19 @@
 # Copyright (c) 2008-2014 Simplistix Ltd
 # See license.txt for license details.
 
+
 class singleton(object):
     def __init__(self, name):
         self.name = name
+
     def __repr__(self):
         return '<%s>' % self.name
     __str__ = __repr__
-    
+
 not_there = singleton('not_there')
 
-from testfixtures.comparison import Comparison, StringComparison, RoundComparison, compare, diff
+from testfixtures.comparison import (Comparison, StringComparison,
+                                     RoundComparison, compare, diff)
 from testfixtures.tdatetime import test_datetime, test_date, test_time
```

Comment:
I must have been lazy at that time, my personal style would indeed be to put one per line.
For this `__init__.py` which is only imports, perhaps it's not justified, although at the same time, there is no other code to read in this file and its nature is to change as you expand the library. For other modules, if they have 30 lines of imports, then they're probably a thousand lines. :)